src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -57,6 +62,40 @@
           + "    CREDENTIAL = [<credential>]\n"
           + ");";
 
+  private static final String createSnapshotTableTemplate =
+      "CREATE EXTERNAL TABLE [<tableName>]\n"
+          + "WITH (\n"
+          + "    LOCATION = '<destinationParquetFile>',\n"
+          + "    DATA_SOURCE = [<destinationDataSourceName>],\n" // metadata container
+          + "    FILE_FORMAT = [<fileFormat>]\n"
+          + ") AS SELECT * FROM\n"
+          + "    OPENROWSET(\n"
+          + "       BULK '<ingestFileName>',\n"
+          + "       DATA_SOURCE = '<ingestFileDataSourceName>',\n"
+          + "       FORMAT = 'parquet') "
```

Comment:
We define `parquet` as `azureResourceConfiguration.getSynapse().getParquetFileFormatName()` in other queries.

Reply:
The `azureResourceConfiguration.getSynapse().getParquetFileFormatName()` is "ParquetFileFormat" and is used for the FILE_FORMAT parameter so it's slightly different (although I agree it's confusing)! Do you think it's clearer to make that a config variable too?

Reply:
Hmm. I'm not sure it needs to be a config variable. It may be better to just hardcode it, since I don't _think_ it will change. @snf2ye what do you think?

Reply:
I don't think it will change. We have this same pattern for 'CSV' in some of the other queries.

Reply:
Why did we make the file format configurable in the intial datasets implementation?

Reply:
It's a different parameter - The parameter set in `getParquetFileFormatName()`is "ParquetFileFormat" and is created as a result of us manually running the following command in synapse:
`CREATE EXTERNAL FILE FORMAT [ParquetFileFormat]
WITH (
   FORMAT_TYPE = PARQUET
)`

Reply:
Looks like this same format is re-used on line 70: <fileFormat>